src/backend/launcher.ts

```diff
@@ -313,7 +307,7 @@ function setupWineEnvVars(gameSettings: GameSettings, gameId = '0') {
   }
 
   if (gameSettings.showFps) {
-    ret.DXVK_HUD = 'fps'
+    isMac ? (ret.MTL_HUD_ENABLED = '1') : (ret.DXVK_HUD = 'fps')
```

Comment:
This is my preference but using ternaries for branching is awkward. I know this is for the assignment of different `ret` fields but I'd do:
```typescript
if (isMac) {
  ret.MTL_HUD_ENABLED = '1'
} else {
  ret.DXVK_HUD = 'fps'
}

```

Reply:
done